makefile

```diff
@@ -40,6 +33,7 @@ man:	config
 	man target/dmenu.1
 
 test:	all
+	cd src && cargo test
```

Comment:
Thanks for adding tests. This fixes the problem with removing `checkdeps.sh`. Now we can be certain that everything works without needing to test it beforehand.